src/styles/_catalog.scss

```diff
@@ -1,6 +1,8 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-width);
+  justify-content: center;
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
```

Comment:
Do you think these styles still matter after applying grid layout?

Reply:
Yes it center Grid container in the page.

Reply:
For my case "justify-content: center" is enoght.
https://youtu.be/66pn_kRNMto?t=92